File: src/week_2_arrays/notes/src/scores/scores2.c
#include "cs50.h"
#include <stdio.h>

// You can declare a global variable that will in scope to every function in this file
//const int N = 3;

int main(void)
{
    // Common way to name constant integers is capitalized it
    const int N = 3;
    int scores[N];

//  Gives you an array with values inside
//  int scores = {72, 73, 33};

    for (int i = 0; i < N; i++)
    {
        scores[i] = get_int("Score: ");
    }

    printf("Average: %f\n", (scores[0] + scores[1] + scores[2]) / (float) N);

    return 0;
}

File: src/week_6_python/notes/src/scores/scores_2.py
from cs50 import get_int

# Create an empty list
scores = []

for i in range(3):
    score = get_int("Score:  ")
    # Add new value onto the end
    scores.append(score)

average = sum(scores) / len(scores)
print(f"Average: {average:.1f}")
